libsolidity/formal/SMTChecker.cpp

```diff
@@ -701,6 +704,19 @@ void SMTChecker::resetVariables(vector<Declaration const*> _variables)
 	}
 }
 
+void SMTChecker::mergeVariables(vector<Declaration const*> _variables, smt::Expression _condition, VariableSequenceCounters& _countersEndTrue, VariableSequenceCounters& _countersEndFalse)
+{
+	for (auto const* decl: _variables)
+	{
+		newValue(*decl);
+		int trueCounter = _countersEndTrue.at(decl);
+		int falseCounter = _countersEndFalse.at(decl);
+		smt::Expression e = currentValue(*decl) == smt::Expression::ite(_condition, valueAtSequence(*decl, trueCounter), valueAtSequence(*decl, falseCounter));
+		m_interface->addAssertion(e);
+		setUnknownValue(*decl);
```

Comment:
Didn't we just set it to a specific, known value?

Reply:
Yea, I'd kept it to have the interval set as well, but we probably don't need it. Removed.